lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.scss

```diff
@@ -67,28 +88,27 @@
     &-metadata-action-buttons {
         display: flex;
         justify-content: space-evenly;
-        margin: 10px;
+
+        &-clear {
+            color: var(--adf-metadata-action-button-clear-color);
+        }
     }
 
     &-metadata-categories-header {
-        display: flex;
-        flex-direction: column;
-        padding: 0 24px;
-
-        .adf-metadata-categories-title {
+        .adf-categories-button {
             display: flex;
             justify-content: space-between;
             align-items: center;
             font-size: 15px;
-            height: 64px;
-
-            button {
-                margin-right: -14px;
-            }
+            height: $panel-properties-height;
 
             [hidden] {
                 visibility: hidden;
             }
         }
     }
 }
+
+.adf-metadata-grouped-properties-container .mat-content.mat-content-hide-toggle {
```

Comment:
Can we avoid styling internal mat classes?

Reply:
removed entirely as seems to not be used/working